x-pack/plugins/observability_solution/slo/server/services/create_slo.ts

```diff
@@ -47,62 +48,41 @@ export class CreateSLO {
 
     const rollbackOperations = [];
 
-    await this.repository.save(slo, { throwOnConflict: true });
-    rollbackOperations.push(() => this.repository.deleteById(slo.id));
+    // can be done async as we are not using the result
+    void this.repository.save(slo, { throwOnConflict: true });
```

Comment:
We are supposed to throw on conflict here so we don't continue with the other operations. Should we not keep that?